test/common/unit/libraries/MessageLib.t.sol

```diff
@@ -113,7 +113,7 @@ contract TestMessageLibIdentities is Test {
         MessageLib.NotifyShareClass memory a = MessageLib.NotifyShareClass({
             poolId: poolId,
             scId: scId,
-            name: name,
+            name: string(abi.encodePacked(name)),
```

Comment:
We have a `CastLib.toString` for this. Although not sure which is faster TBH.

Reply:
Since it's in test code, don't think it really matters

Reply:
Ah right! correct